sota-implementations/ddpg/ddpg.py

```diff
@@ -108,7 +108,7 @@ def main(cfg: "DictConfig"):  # noqa: F821
     for _, tensordict in enumerate(collector):
         sampling_time = time.time() - sampling_start
         # Update exploration policy
-        exploration_policy.step(tensordict.numel())
+        exploration_policy[1].step(tensordict.numel())
```

Comment:
I'm not completely sure if this is the best way to do this. Does there happen to be some alternative to `TensorDictSequential` which does essentially the same thing but also provides a `step` function?

Reply:
Well, it looks like the same thing was done when `EGreedyWrapper` was updated to `EGreedyModule`, so I guess it's alright:

https://github.com/pytorch/rl/blob/bdc97846dbcaffbd62da8f24aecce437fde9f60b/sota-implementations/bandits/dqn.py#L89-L97
https://github.com/pytorch/rl/blob/bdc97846dbcaffbd62da8f24aecce437fde9f60b/sota-implementations/bandits/dqn.py#L125

Reply:
Yep either that or

```python
def update_exploration(module):
    if isinstance(module, ExplorationModule):
        module.set()
policy.apply(update_exploration)
```
We could make sure that all exploration modules have the same parent class and use that update function across examples.